code/hispania/modules/mob/living/carbon/human/species/monkey_combat.dm

```diff
@@ -206,6 +206,14 @@
 				mode = MONKEY_FLEE
 				return TRUE*/
 
+			// nuh uh you don't pull me!
+			if(pulledby || prob(MONKEY_PULL_AGGRO_PROB))
```

Comment:
HICISTES LO CONTRARIO.

Reply:
EH? LO PUSE EN MONKEY HUNT

Reply:
ESTAN EN MONKEY HUNT CIEGO

Reply:
Ya, tiene mas sentido. Me asuste, perdón

Ahora que el intent vuelva a help cuando se calmen.

Reply:
![image](https://user-images.githubusercontent.com/49106866/116820635-11d17e80-ab44-11eb-8b40-931c2565f431.png)
deberia estar implementado lo del intent help

Reply:
Hmmm, raro una vez in-game no habia pasado eso. Sera que estaba agresivo con alguien que no veía.

Reply:
debo decir que esto es un poco diferente de la logica anterior, antes si el mono estaba molesto inmediatamente activaba este efecato, ahora si el mono está molesto tiene un probabilidad de activar el efecto.